crates/fluvio-sc/Cargo.toml

```diff
@@ -47,6 +47,7 @@ tracing = { workspace = true }
 
 
 # Fluvio dependencies
+fluvio = { workspace = true }
```

Comment:
Is too bad to add this dependence? 
We need `DomainConnector::try_from`, I even tried to move it to fluvio-auth for example, but this method needs the build features of fluvio package.

Reply:
If just need `DomainConnector` this can be imported from `fluvio-future`

Reply:
True, but we need `fluvio::config::TlsPolicy` there too. 
Same problem, this is hard to move because use build features of fluvio package.